scripts/ps.py

```diff
@@ -17,17 +17,28 @@
 
 import psutil
 
+PROC_STATUSES_RAW = {
+    psutil.STATUS_RUNNING: "R",
+    psutil.STATUS_SLEEPING: "S",
+    psutil.STATUS_DISK_SLEEP: "D",
+    psutil.STATUS_STOPPED: "T",
+    psutil.STATUS_TRACING_STOP: "t",
+    psutil.STATUS_ZOMBIE: "Z",
+    psutil.STATUS_DEAD: "X",
+    psutil.STATUS_WAKING: "W"
+}
```

Comment:
No we shouldn't. Come up with some letter. `psutil.STATUS_WAKE_KILL` can be mapped to `"WK" and `psutil.STATUS_WAKING` could be `"WA"`.